include/uv.h

```diff
@@ -1242,6 +1242,7 @@ typedef enum {
   UV_FS_FTRUNCATE,
   UV_FS_UTIME,
   UV_FS_FUTIME,
+  UV_FS_LUTIME,
```

Comment:
This breaks ABI (subsequent enums get renumbered.) Can you move this to the end?